### PR TITLE
Allows exterior turfs to show atmos overlays.

### PR DESCRIPTION
**Pull request by MistakeNot4892**

### CODE REVIEW

**Review by comma — DISMISSED** [review]

**Comment by afterthought2** [comment]
Conflicts

**Comment by MistakeNot4892** [comment]
Conflicts resolved.

File: code/game/turfs/exterior/_exterior.dm [hunk 1]
```diff
@@ -60,8 +60,12 @@
 		set_light(owner.lightlevel, 0.1, 2)
 		if(owner.planetary_area && istype(loc, world.area))
 			ChangeArea(src, owner.planetary_area)
-	update_icon(TRUE)
+	..()
+	. = INITIALIZE_HINT_LATELOAD
+
+/turf/exterior/LateInitialize()
 	. = ..()
+	update_icon(TRUE)
```
Comment (Lohikar) [inline code/game/turfs/exterior/_exterior.dm:68]: Couldn't you do this by calling queue icon update in Initialize instead?

Comment (MistakeNot4892) [reply code/game/turfs/exterior/_exterior.dm:68]: The argument needs to be passed and it's important that it happens late, I wasn't aware I could give arguments to the queue proc.

**Review by comma — APPROVED** [review]

**Pull request merged**